ios-base/AppDelegate.swift

```diff
@@ -52,6 +53,16 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
       window?.rootViewController = UIStoryboard(name: "Main", bundle: nil).instantiateInitialViewController()
     }
   }
+
+  private func setupFirebase() {
```

Comment:
I'd wrap every interaction with Firebase in a `AnalyticsManager`, that way you'd only import Firebase there. In the app delegate you'd only call `AnalyticManager.shared.setup()`.
When firing analytics calls you'd do `AnalyticManager.shared.goToProfileAction()` that way if someday we are switching the analytics framework we'd only need to do changes in the manager and not in every single call all over the code base.

Reply:
Yep, that's the idea. But it's coming in a separate PR 😄 this is just the basic firebase+crashlytics setup.

My plan for the analytics was to have a Manager as you said.